packages/vite-plugin-cloudflare/src/index.ts

```diff
@@ -328,12 +328,12 @@ export function cloudflare(pluginConfig: PluginConfig = {}): vite.Plugin[] {
 					"Unexpected error: No Vite HTTP server"
 				);
 
-				if (!miniflare) {
-					const inputInspectorPort = await getInputInspectorPortOption(
-						pluginConfig,
-						viteDevServer
-					);
+				const inputInspectorPort = await getInputInspectorPortOption(
+					pluginConfig,
+					viteDevServer
+				);
```

Comment:
Doesn't this mean that, in the default case, a new port is allocated on every restart because the existing one is already in use?

Reply:
No, because I modified `getInputInspectorPortOption` to reuse the already existing port if there is one 🙂